### PR TITLE
Add method, status and format tags to metrics

### PR DESCRIPTION
**Pull request by ChrisBr**
These values are provided by the payload anyway
and will allow very interesting queries like

* group / count only successfull requests
* group / count only GET requests
* group / count only JSON requests

I think these tags should be included by default because they're potentially interesting for all users.

### CODE REVIEW

**Comment by ChrisBr** [comment]
We could also include ``path`` to partly solve #30.

File: lib/influxdb-rails.rb [hunk 1]
```diff
@@ -84,6 +84,10 @@ def report_exception(e, env = {})
       def handle_action_controller_metrics(_name, start, finish, _id, payload)
         tags = {
           method:   "#{payload[:controller]}##{payload[:action]}",
+          status:   payload[:status],
+          format:   payload[:format],
+          http_method:   payload[:method],
+          path:     payload[:path],
```
Comment (dmke) [inline lib/influxdb-rails.rb]: Please keep values aligned.

File: spec/unit/influxdb_rails_spec.rb [hunk 2]
```diff
@@ -12,16 +12,20 @@
...
         },
         tags: {
           method: 'MyController#show',
+          status: 200,
+          format: '*/*',
+          http_method: 'GET',
+          path: '/posts',
           server: Socket.gethostname,
-          app_name: 'my-rails-app',
+          app_name: 'my-rails-app'
```
Comment (dmke) [inline spec/unit/influxdb_rails_spec.rb]: Same here, please align values. Also keep the trailing comma.

**Review by dmke — CHANGES_REQUESTED** [review]
Some styling issues, but looks good.

**Comment by ChrisBr** [comment]
@dmke thanks for the feedback, updated!

**Review by dmke — APPROVED** [review]

**Comment by dmke** [comment]
Thanks, I'll merge it later when I'm back home.

**Comment by ChrisBr** [comment]
> Thanks, I'll merge it later when I'm back home.
 
Can we expect a new version release soon?

**Comment by dmke** [comment]
Immediately after. :-)

**Comment by ChrisBr** [comment]
> Immediately after. :-)

Awesome :+1: Thanks for your great work :heart: 

Btw, is there somewhere documentation etc for creating grafana dashboards out of this data? We still struggle to create a good dashboard ...

**Comment by dmke** [comment]
I've merged your changes into a 1.0-beta branch (78d39f8...ed785c7), because I forgot that I have a pending 1.0 release.

So far, Travis looks good (https://travis-ci.org/influxdata/influxdb-rails/builds/458571410). When this finishes, I'll issue a beta release.

**Pull request closed without merging**